examples/cpp/pyperf/PyPerfCollapsedPrinter.cc

```diff
@@ -75,25 +75,25 @@ void PyPerfCollapsedPrinter::processSamples(
   auto symbols = util->getSymbolMapping();
   for (auto& sample : samples) {
     int frames = 0;
-    std::fprintf(output_file, "%s-%d/%d", sample.comm.c_str(), sample.pid, sample.tid);
+    std::fprintf(output_file, "%s-%d/%d_[p]", sample.comm.c_str(), sample.pid, sample.tid);
```

Comment:
This is the first frame of the stack (process name). We currently don't parse annotations from it, I think (and neither does e.g `flamegraph.pl`). So drop it for now and we can first decide if we want to add it.

Anyway, if we do decide to add it - we need to modify gProfiler as well, because it parses the TID/PID from that frame.